src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java

```diff
@@ -0,0 +1,104 @@
+
+        //Act
+        String[] executionCommand = executor.createExecutionCommand();
+        List<String> commandAsList = Arrays.asList(executionCommand);
+
+        //Assert
+        String actualLatestCommit = findValueInArray(commandAsList, "-latestCommit");
+        String actualPullRequestNumber = findValueInArray(commandAsList, "-pullRequestNumber");
+        String actualRepoUrl = findValueInArray(commandAsList, "-repoUrl");
+        String actualTargetBranch = findValueInArray(commandAsList, "-targetBranch");
+        Assert.assertEquals(latestCommit, actualLatestCommit);
+        Assert.assertEquals(pullRequestNumber, actualPullRequestNumber);
+        Assert.assertEquals(repoUrl, actualRepoUrl);
+        Assert.assertEquals(targetBranch, actualTargetBranch);
+
+    }
+
+    @Test
+    public void execute_hasTechOptions_shouldAdd() throws IOException {
+        //Arrange
+        String packagesIncluded = "io.include.*";
+        String packagesExcluded = "io.exclude.*";
+        DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions = new DescribableList<>(JavaOptions.DescriptorImpl.NOOP);
+        technologyOptions.add(new JavaOptions(packagesIncluded, packagesExcluded));
+        PrConfigCommandExecutor executor = createTestedExecutor(technologyOptions);
+
+        //Act
+        String[] executionCommand = executor.createExecutionCommand();
+        List<String> commandAsList = Arrays.asList(executionCommand);
+
+        // Assert
+        String actualIncluded = findValueInArray(commandAsList, "-packagesincluded");
+        String actualExcluded = findValueInArray(commandAsList, "-packagesexcluded");
+        Assert.assertEquals(packagesIncluded, actualIncluded);
+        Assert.assertEquals(packagesExcluded, actualExcluded);
+
+    }
+
+    private PrConfigCommandExecutor createTestedExecutor(DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions) throws IOException {
+        BaseCommandArguments baseCommandArguments = createBaseCommandArguments();
+        PrConfigCommandArguments prConfigCommandArguments = createPrConfigArguments(technologyOptions);
+        PrConfigCommandExecutor executor = new PrConfigCommandExecutor(nullLogger, baseCommandArguments,
+                prConfigCommandArguments);
+        executor.setJenkinsUtils(createMockJenkinsUtils());
+        Runtime runtimeMock = mock(Runtime.class);
+
+        //Act
+        executor.setRuntime(runtimeMock);
+        return executor;
+    }
+
+    private PrConfigCommandArguments createPrConfigArguments(DescribableList<TechnologyOptions,
+     TechnologyOptionsDescriptor> technologyOptions) {
+        PrConfigCommandArguments configArguments = new PrConfigCommandArguments(technologyOptions, latestCommit,
+                pullRequestNumber, repoUrl, targetBranch);
+
+        return configArguments;
+    }
+
+    private DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> createTechnologyOptions() {

```

Comment:
Seems like no one is using this code. Am I'm missing something?

Reply:
Done